### DISCUSSION TITLE
Can we include hidden files in the file picker?

### DISCUSSION THREAD

**Discussion by slinlee** [helix-editor/helix#1820] [Q&A] [answered]
The file picker default behavior is really nice, especially how it honors `.gitignore` when you're in a git project.

I was wondering if there's a way to have it include hidden files in the file picker though. This includes the file `.gitignore` or directories like `.github/workflows/xyz.yml`

Right now I open those manually with `:o .gitignore` which at least has auto-completion. 👍🏽

**Comment by sudormrfbin** [comment, marked as answer]
It's configurable using the `editor.file-picker.hidden` - https://docs.helix-editor.com/configuration.html#editorfile-picker-section.

**Reply by slinlee** [reply]
Thanks. Apparently I didn't know to click on that level of the TOC in the docs. My bad :)

**Comment by hunger** [comment]
Some short-term way to show hidden files for the current "open file" operation would be nice to have! Not having hidden files mess up the search result is the correct thing in 95% of the searches I do, so I am reluctant to change the default. It would be awfully convenient to be able to just toggle the "hidden" setting for the one search I am doing in the last 5%.
